### PR TITLE
Fix Ubuntu Installation

### LINKED ISSUE #81: Failed to start chronyd.service: Unit chronyd.service not found.
image

There us an error in the skript:

This is the fix:
apt install chrony 

pls add it to the skript

### LINKED ISSUE #82: Ubuntu 20.04 needs elseif because python is now phyton 3
```bash

# Install necessary aptitudes for sinusbot.

magentaMessage "Installing necessary packages. Please wait..."

if [[ -f /etc/centos-release ]]; then
  yum -y -q install screen xvfb libxcursor1 ca-certificates bzip2 psmisc libglib2.0-0 less cron-apt ntp python iproute which dbus libnss3 libegl1-mesa x11-xkb-utils libasound2 libxcomposite-dev libxi6 libpci3 libxslt1.1 libxkbcommon0 libxss1 >/dev/null
  update-ca-trust extract >/dev/null
else
  # Detect if systemctl is available then use systemd as start script. Otherwise use init.d
  if [ "$OSRELEASE" == "18.04" ] && [ "$OS" == "ubuntu" ]; then
    apt-get -y install chrony
  else
    apt-get -y install ntp
  fi
  apt-get -y -qq install libfontconfig libxtst6 screen xvfb libxcursor1 ca-certificates bzip2 psmisc libglib2.0-0 less cron-apt **python** iproute2 dbus libnss3 libegl1-mesa x11-xkb-utils libasound2 libxcomposite-dev libxi6 libpci3 libxslt1.1 libxkbcommon0 libxss1
  update-ca-certificates >/dev/null
fi
...
```
must be named as phyton3 in Ubuntu 20.04 Focal fassa. By applying this change sinunsbot works like a charm again.

### LINKED ISSUE #91: Failed to set ntp: NTP not supported
Trigger für man-db (2.9.4-2) werden verarbeitet ...
Trigger für dbus (1.12.20-2) werden verarbeitet ...
Packages installed!
Failed to set ntp: NTP not supported
Command which failed was: "      timedatectl set-ntp yes". Please try to execute it manually and attach the output to the bug report in the forum thread.

Executing the command manually also results in "Failed to set ntp: NTP not supported".
(Debian 11 latest version)

### LINKED ISSUE #97: Installer on Ubuntu 22 not working
Added the testing matrix in https://github.com/SinusBot/installer-linux/pull/96 so we see the failure.

Most likely related to:

- https://github.com/SinusBot/installer-linux/issues/91
- #82 
- #81

cc @Lulalaby

### PR DESCRIPTION
**Pull request by Lulalaby**
Closes: #97
Closes: #91
Closes: #82
Closes: #81

Follow-up of #98, by request of @mxschmitt

### CODE REVIEW

**Comment by Lulalaby** [comment]
Merge #100 before this

**Comment by Lulalaby** [comment]
https://github.com/Lulalaby/sinusbot-installer-linux/actions/runs/3195918910/jobs/5217170957

**Comment by adiantek** [comment]
Still doesn't work on Ubuntu 22.04 (image from http://download.proxmox.com/images/system/ubuntu-22.04-standard_22.04-1_amd64.tar.zst, OpenVZ):
```
Downloading YT-DL now...
Download successful!
/usr/bin/env: ‘python’: No such file or directory
Command which failed was: "      greenMessage "config.ini created successfully."". Please try to execute it manually and attach the output to the bug report in the forum thread.
If it still doesn't work report this to the author at https://forum.sinusbot.com/threads/sinusbot-installer-script.1200/ only. Not a PN or a bad review, cause this is an error of your system not of the installer script. Line 896.
```
I need to `apt install python-is-python3`


------
Another error:
```

lis 09 01:12:05 sinusbot sinusbot[154]: 2022/11/09 01:12:05 9a5794a5 444b63eb DEBUG  TS>/opt/sinusbot/teamspeak3-client/ts3client_linux_amd64: error while loading shared libraries: libXdamage.so.1: cannot open shared object file: No such file or directory
lis 09 01:12:05 sinusbot sinusbot[154]: 2022/11/09 01:12:05 9a5794a5 444b63eb DEBUG  Closed.
lis 09 01:12:05 sinusbot sinusbot[154]: 2022/11/09 01:12:05 9a5794a5 444b63eb INFO   TSClient quit. LogLevel has been increased, please try to connect again to see more details.
```
`apt install libxdamage1` solved it.

File: sinusbot_installer.sh [hunk 1]
```diff
@@ -621,76 +608,31 @@ fi
...
-    apt-get -y install ntp
-  fi
-  apt-get install -y -qq --no-install-recommends libfontconfig libxtst6 screen xvfb libxcursor1 ca-certificates bzip2 psmisc libglib2.0-0 less python iproute2 dbus libnss3 libegl1-mesa x11-xkb-utils libasound2 libxcomposite-dev libxi6 libpci3 libxslt1.1 libxkbcommon0 libxss1
+  apt-get install -y -qq --no-install-recommends libfontconfig libxtst6 screen xvfb libxcursor1 ca-certificates bzip2 psmisc libglib2.0-0 less python3 iproute2 dbus libnss3 libegl1-mesa x11-xkb-utils libasound2 libxcomposite-dev libxi6 libpci3 libxslt1.1 libxkbcommon0 libxss1
   update-ca-certificates >/dev/null
 fi
-
+if [[ $(cat /etc/*release | grep "PRETTY_NAME=" | sed 's/PRETTY_NAME=//g') =~ "Debian" ]]; then
+    apt-get install -y -qq --no-install-recommends python-is-python3
+fi
```
Comment (Lulalaby) [inline sinusbot_installer.sh:619]:
```suggestion
if [[ $(cat /etc/*release | grep "PRETTY_NAME=" | sed 's/PRETTY_NAME=//g') =~ "Debian" ]]; then
    apt-get install -y -qq --no-install-recommends python-is-python3
fi
if [[ $(cat /etc/*release | grep "PRETTY_NAME=" | sed 's/PRETTY_NAME=//g') =~ "Ubuntu" ]]; then
    apt-get install -y -qq --no-install-recommends python-is-python3
fi
```

File: sinusbot_installer.sh [hunk 2]
```diff
@@ -621,76 +608,31 @@ fi
...
   update-ca-trust extract >/dev/null
 else
-  # Detect if systemctl is available then use systemd as start script. Otherwise use init.d
-  if [ "$OSRELEASE" == "18.04" ] && [ "$OS" == "ubuntu" ]; then
-    apt-get -y install chrony
-  else
-    apt-get -y install ntp
-  fi
-  apt-get install -y -qq --no-install-recommends libfontconfig libxtst6 screen xvfb libxcursor1 ca-certificates bzip2 psmisc libglib2.0-0 less python iproute2 dbus libnss3 libegl1-mesa x11-xkb-utils libasound2 libxcomposite-dev libxi6 libpci3 libxslt1.1 libxkbcommon0 libxss1
+  apt-get install -y -qq --no-install-recommends libfontconfig libxtst6 screen xvfb libxcursor1 ca-certificates bzip2 psmisc libglib2.0-0 less python3 iproute2 dbus libnss3 libegl1-mesa x11-xkb-utils libasound2 libxcomposite-dev libxi6 libpci3 libxslt1.1 libxkbcommon0 libxss1
```
Comment (Lulalaby) [inline sinusbot_installer.sh]:
```suggestion
  apt-get install -y -qq --no-install-recommends libfontconfig libxtst6 screen xvfb libxcursor1 ca-certificates bzip2 psmisc libglib2.0-0 less python3 iproute2 dbus libnss3 libegl1-mesa x11-xkb-utils libasound2 libxcomposite-dev libxi6 libpci3 libxslt1.1 libxkbcommon0 libxss1 libxdamage1
```

File: sinusbot_installer.sh (hunk 1 above)
Comment (Lulalaby) [reply sinusbot_installer.sh:619]: mhh have to filter for old ubuntu version

**Comment by Lulalaby** [comment]
tbh I don't think i can fix this. Sorry

**Pull request closed without merging**